citadel.dme

```diff
@@ -14,6 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_mapload\_basemap.dm"
+#include "_mapload\tether.dm"
```

Comment:
uhhhh

Reply:
yeah i know this is intentional because i was testing